api/appointments.ts

```diff
@@ -3,13 +3,13 @@ import db from './db';
 import { QueryResult } from 'pg';
 
 export default handler(async (request: any) => {
-  let connection = db();
-  let results: QueryResult;
-  // TODO: Revise Query String to utilize Bar's Query.
-  if (request.query.id === undefined) {
-    results = await connection.query('SELECT * FROM appointments');
-  } else {
-    results = await connection.query('SELECT * FROM appointments WHERE appointments.appointmentid = $1', [request.query.id]);
-  }
-  return results.rows;
+    return await db(async (connection) => {
```

Comment:
`await` here is unnecessary; you can just return the Promise.
Likewise, since this is a single statement lambda, you can omit the `{` and `return`.
Thus you get the concise form: `export default handler((request) => db(async (connection) => { ... }));`
No functional difference though.